### PR TITLE
Refactor OGC service and init CSW

### PR DESCRIPTION
**Pull request by mbarbet**

### CODE REVIEW

File: src/main/java/io/arlas/server/app/OGCConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package io.arlas.server.app;
```
Comment (sfalquier) [inline src/main/java/io/arlas/server/app/OGCConfiguration.java]: Disclaimer is missing :o(

File: src/main/java/io/arlas/server/ogc/common/utils/CSWConstant.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package io.arlas.server.ogc.common.utils;
```
Comment (sfalquier) [inline src/main/java/io/arlas/server/ogc/common/utils/CSWConstant.java]: Disclaimer is missing :o(

File: src/main/java/io/arlas/server/ogc/csw/utils/CSWRequestType.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package io.arlas.server.ogc.csw.utils;
```
Comment (sfalquier) [inline src/main/java/io/arlas/server/ogc/csw/utils/CSWRequestType.java]: Disclaimer is missing :o(

File: src/main/java/io/arlas/server/app/OGCConfiguration.java (hunk 1 above)
Comment (mbarbet) [reply src/main/java/io/arlas/server/app/OGCConfiguration.java]: Fix

File: src/main/java/io/arlas/server/ogc/common/utils/CSWConstant.java (hunk 2 above)
Comment (mbarbet) [reply src/main/java/io/arlas/server/ogc/common/utils/CSWConstant.java]: Fix

File: src/main/java/io/arlas/server/ogc/csw/utils/CSWRequestType.java (hunk 3 above)
Comment (mbarbet) [reply src/main/java/io/arlas/server/ogc/csw/utils/CSWRequestType.java]: Fix

**Pull request merged**